### DISCUSSION TITLE
接入mysql5.1.6的时候报错了

### DISCUSSION THREAD

**Discussion by ChenHuanNet** [dotnetcore/FreeSql#1313] [General]
5.1.6没有 这个函数 @@lower_case_table_names，

**Comment by 2881099** [comment]
具体做什么操作的时候报错

**Reply by ChenHuanNet** [reply]
就连接数据库，查询就报错，
![image](https://user-images.githubusercontent.com/48199389/198527901-dfb5e41c-048a-4297-848e-9d3119da22cc.png)

![image](https://user-images.githubusercontent.com/48199389/198527725-3facd75a-6059-4e35-b1da-0c5ccea2f931.png)

![image](https://user-images.githubusercontent.com/48199389/198528313-6118b85d-999d-47b9-80f4-a1de7fa41e75.png)

![image](https://user-images.githubusercontent.com/48199389/198528914-6b87049b-9b80-4283-9e3a-8b27f6167113.png)

![image](https://user-images.githubusercontent.com/48199389/198528985-da12e3da-b40d-49f5-95ee-aab7b9f93168.png)

**Reply by ChenHuanNet** [reply]
在高版本的mysql上面就没问题，我猜测是低版本没有这个@@lower_case_table_names 函数导致的，但是有没有办法让ORM兼容他，或者哪里设置下，可以跳过它，不然我需要更换ORM来做项目很麻烦。。客户的mysql很老旧

**Reply by ChenHuanNet** [reply]
我尝试使用过EF Core去测读取数据，发现是可以读到的。。大佬帮忙解决下，就不用换ORM了。。

**Reply by 2881099** [reply]
freesql 没用过这个函数，应该是 mysql.data.dll 驱动使用了导致错误。

你试试换成 freesql.provider.mysqlconnector.dl

具体测试直接使用 new MySqlConnection(str).Open()，如果不报错就成功了。

**Reply by 2881099** [reply]
记住是，换，不要 freesql.provider.mysql.dll 和 freesql.provider.mysqlconnector.dll 两个同时引用

**Reply by ChenHuanNet** [reply]
OK，我测试一下。之前是用freesql.provider.mysql.dll
